modules/gateways/Blockonomics/Blockonomics.php

```diff
@@ -79,19 +79,55 @@ private function generateCallbackSecret() {
 	 */
 	public function getApiKey() {
 		return Capsule::table('tblpaymentgateways')
-			->where('gateway', 'blockonomics')
-			->where('setting', 'ApiKey')
-			->value('value');
+				->where('gateway', 'blockonomics')
+				->where('setting', 'ApiKey')
+				->value('value');
 	}
 
 	/*
-	 * Get user configured API key from database
+	 * Get list of crypto currencies supported by Blockonomics
 	 */
-	public function getAltcoins() {
-		return Capsule::table('tblpaymentgateways')
-			->where('gateway', 'blockonomics')
-			->where('setting', 'Altcoins')
-			->value('value');
+	public function getSupportedCurrencies() {
```

Comment:
This can return a dict() so that a caller that can iterating over array to match code

Reply:
Fixed with 5138aa5